src/development/going-live.md

```diff
@@ -1,371 +1,139 @@
+Configure your DNS provider to point your domain to your Platform.sh Master environment.
 
-```bash
-openssl rsa -in server.pass.key -out server.key
-```
+The way to do so will vary somewhat depending on your registrar, but nearly all registrars should allow you to set a CNAME.  Some will call it an Alias or similar alternate name, but either way the intent is to say "this domain should always resolve to... this other domain".  Add a CNAME record from your desired domain (`www.example.com`) to the master environment hostname you wrote down earlier.
 
-You now have a `server.key` private key file in your current working
-directory.
+If you have multiple domains you want to be served by the same application you will need to add a CNAME record for each of them.
 
-### Generate CSR
+Note that depending on your registrar and the TTL you set, it could take anywhere from 15 minutes to 12 hours for the DNS change to fully propagate across the Internet.
 
-A CSR is a certificate signing request and is also required when
-purchasing an SSL cert. Using the private key from the previous step,
-generate the CSR. This will require you to enter identifying information
-about your organization and domain.
+### Apex domains
 
-Though most fields are self-explanatory, pay close attention to the
-following:
+One of the challenges of using a cloud hosting service like Amazon Web Services (AWS) Elastic Cloud (EC2) is that you need to point your DNS to a CNAME. The problem is the DNS RFC (RFC1033) requires the "zone apex" (sometimes called the "root domain" or "naked domain") to be an "A Record," not a CNAME. Some DNS providers allow you to do so anyway while others do not. Some provide an alternative means of registering an indirect apex domain.
 
-* Country Name: The two letter code, in ISO 3166-1 format, of the country
-in which your organization is based.
+In other words, with many DNS providers:
 
-* Common Name: This is the fully qualified domain name that you wish to
-secure.
-  - For a single subdomain: `www.example.com`
-  - For all subdomains, specify the wildcard URL: `*.example.com`
-  - For the root domain only: `example.com`
+----------------- ------------------- ------ ----------------------------------------------
+you can do this   `www.domain.tld`    CNAME  `<environment>-<project>.<region>.platform.sh`
+you can't do this `domain.tld`        CNAME  `<environment>-<project>.<region>.platform.sh`
+----------------- ------------------- ------ ----------------------------------------------
 
-  The Common Name field must match the secure domain. You cannot purchase
-a certificate for the root domain, e.g. `example.com`, and expect to
-secure `www.example.com`. The inverse is also true. Each domain can have
-one certificate and it can be attached to the main domain or subdomain.
+However, using an A Record is not possible as the IP address of the server may change from time to time, especially with frequent redeployments as in Platform.sh's case.
```

Comment:
I don't get this ... The IP address will be those on our gateways. They are considered stable. And, some DNS provider supports adding multiple 'A' records and load balance them. Customer may consider using this kind of 'A' records when 'CNAME' record is not applicable.

On the other hand, some DNS provider supports HTTP/HTTPS redirect on apex domain. That can be used as well when 'CNAME' is not applicable.